resources/object_repo/installed_operators_obj.resource

```diff
@@ -6,7 +6,7 @@ Object Repository for Installed Operators Screen
 ${elemDbaasStatus}          //a[contains(.,"OpenShift Database Access Operator")]//ancestor::tr/td[contains(.,"Succeeded")]
 ${titleDbaasOperator}       //h1[contains(.,"OpenShift Database Access Operator") and contains(@class,"logo__name")]
 ${elemDbaasInvStatus}       //td[contains(.,"<<paname>>")]/parent::tr/td[contains(.,"SpecSynced, InventoryReady")]
-${titleDbaasInventories}    //h1[contains(.,"DBaaSInventories")]
+${titleDbaasInventories}    //h1[contains(.,"DBaaSInventorys")]
```

Comment:
IMHO, this one should not be merged as we have defect for [DBAAS-791](https://issues.redhat.com/browse/DBAAS-791)